src/lib/logical_query_plan/jit_aware_lqp_translator.hpp

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "lqp_translator.hpp"
+#include "operators/jit_operator.hpp"
+#include "operators/jit_operator/operators/jit_compute.hpp"
+
+namespace opossum {
+
+/* This class can be used as a drop-in replacement for the LQPTranslator.
+ * The JitAwareLQPTranslator will try to translate multiple AbstractLQPNodes into a single JitOperator, whenever
+ * that is possible and seems beneficial. Otherwise it will fall back to the LQPTranslator.
```

Comment:
I believe we already started a discussion on this and think we should discuss this in person together with @mrzzzrm. Adding a second translator with unclear responsiblities /will/ lead to inconsistencies one day.